Driverless AI/getting-started-with-driverless-ai-test-drive/test-drive.md

```diff
@@ -77,16 +77,26 @@ If you are new to Driverless AI we highly recommend you start with the **Test Dr
 
 ## How to Start and End the Test Drive Lab Session
 
-After a successful login the Driverless AI Test Drive Lab will appear:
+After a successful login, select the "View Details" button on lab one, as shown in the image below. Ignore other labs as we will only be working on lab one. 
+
+![lab 1](assets/lab1.jpg)
```

Comment:
We need to name the images so that they are more descriptive.

Please update this image to:

![dai-test-drive-lab1](assets/dai-test-drive-lab1.jpg)

Reply:
I can't see the image.

Reply:
Nvm. The image was renamed to "selecting-Lab-1-Driverless-AI-Test-Drive.jpg"